wordpress-mobile/WordPress-Android
# Jetpack App Basics: Base colors

Fixes #14298 

This PR updates base colors for the Jetpack App
- Updated `styles.xml` to extract primary/secondary colors to become placeholders.
- Updated `colors.xml` for WordPress app.
- Added `colors.xml` for Jetpack app.
- Added `themes.xml` for the WordPress Login library.
- Added the related values directories for various device configuration (values-night, values-v23, values-v27).

**Merge instructions**
- Ensure that PR #14320 & PR #14321 have been merged
- Remove "Not ready for merge" label
- Merge as normal

**Pre test**
- You must update your local copy of `google-services.json` before running the app.
-- Copy existing `org.wordress.android`,  `org.wordress.android.prealpha`, `org.wordress.android.beta` sections and replace with `com.jetpack.android`, `com.jetpack.android.prealpha`, `com.jetpack.android.beta`.

**To Test**
- Build and launch each product flavor. 
- - `wordpressVanillaDebug` (Prod)
- `wordpressWasabiDebug` (Beta)
- `wordpressJalapenoDebug` (Alpha)
- `jetpackVanillaDebug` (Prod)
- `jetpackWasabiDebug` (Beta)
- `jetpackJalapenoDebug` (Alpha)
- Verify that Jetpack build variants show "green" and Wordpress as "Pinkish"

Note that:
- `jetpackDebug` should build all `debug` product flavors related to the `jetpack` dimension.
- `jetpackJalapeno` should build both, the `debug` and `release`, `jalapeno` product flavor for `jetpack` dimension.
- `jetpackWasabi` should build both, the `debug` and `release`, `wasabi` product flavor for `jetpack` dimension.


|Wordpress|Jetpack
|---|---
|![WP](https://user-images.githubusercontent.com/506707/112050334-1d825c00-8b27-11eb-9686-d568631b2926.png)|![JP](https://user-images.githubusercontent.com/506707/112050343-1fe4b600-8b27-11eb-8db4-0d7bb8159b7d.png)
PR submission checklist:
